README.md

```diff
@@ -273,6 +273,40 @@ scripts/delete-test-properties
 
 ### Your Access Token
 
+Here we provide instructions on two ways that you can retrieve your Access Token for use with the `reactor-sdk` project.
+
+#### Programmatically via Adobe's [jwt-auth](https://github.com/adobe/jwt-auth) project.
+
+```javascript
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const jwtAuth = require('@adobe/jwt-auth');
+const Reactor = require('@adobe/reactor-sdk').default;
+
+(async function() {
+  const orgId = process.env['ORG_ID'];
+  // jwt-auth config object: https://github.com/adobe/jwt-auth#config-object
+  const config = {
+    orgId,
+    clientId: 'YOUR_CLIENT_ID',
+    technicalAccountId: "YOUR_TECHNICAL_ACCOUNT_ID",
+    clientSecret: "YOUR_CLIENT_SECRET",
+    metaScopes: ["YOUR_META_SCOPES"],
```

Comment:
Do you think it would be useful to add this meta scopes link somewhere? https://www.adobe.io/developer-console/docs/guides/authentication/JWT/Scopes/

Or is having the jwt-auth repo link enough?